components/first_time/first_time_component.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react';
+import { Text, View, AppRegistry, StyleSheet, TextInput, ScrollView, Button, TouchableOpacity } from 'react-native';
```

Comment:
John and I have started using [Native Base](docs.nativebase.io) as a UI library replacement for React Native's library, and it's super sweet. I would recommend checking it out, and re-writing the UI for your component in that. I don't think doing that is a super high priority, but that's more of a call on @michael-beaver's end.